js/jquery.mapael.js

```diff
@@ -55,6 +64,9 @@
                 , zoomCenterX = 0
                 , zoomCenterY = 0
                 , previousPinchDist = 0;
+                
+            // add plugin class name on element
+            $self.addClass(pluginName);
```

Comment:
Hello,

Thank for the explanation, it's ok for me. However, regarding the second argument, currently, nothing prevent the user from adding a custom class name on the container in order to allow specific selectivity in the CSS, this is why I was curious about the purpose of the class.